### PR TITLE
Casmpet 5414

### PR DESCRIPTION
**Pull request by dzou-hpe**
## Summary and Scope

_Summarize what has changed. Explain why this PR is necessary. What is impacted? Is this a new feature, critical bug fix, etc?_

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

## Issues and Related PRs

[CASMPET-5414](https://jira-pro.its.hpecorp.net:8443/browse/CASMPET-5414)

* Resolves [issue id](issue link)
* Change will also be needed in `<insert branch name here>`
* Future work required by [issue id](issue link)
* Documentation changes required in [issue id](issue link)
* Merge with/before/after `<insert PR URL here>`

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/node_management/Rebuild_NCNs.md [hunk 1]
```diff
@@ -42,8 +42,14 @@ ncn# echo $XNAME
 
 Only follow the steps in the section for the node type that is being rebuilt:
 
-* [Worker node](Rebuild_NCNs/Prepare_Worker_Nodes.md)
-* [Master node](Rebuild_NCNs/Prepare_Master_Nodes.md)
+* [Worker node]
```
Comment (nrockershousen) [inline operations/node_management/Rebuild_NCNs.md:45]: Remove the brackets if no link is being used.

File: operations/node_management/Rebuild_NCNs.md [hunk 2]
```diff
@@ -42,8 +42,14 @@ ncn# echo $XNAME
...
+```
```
Comment (nrockershousen) [inline operations/node_management/Rebuild_NCNs.md:46]: indent under the bullet point

File: operations/node_management/Rebuild_NCNs.md [hunk 3]
```diff
@@ -42,8 +42,14 @@ ncn# echo $XNAME
...
+ncn-m001:# /usr/share/doc/csm/upgrade/1.2/scripts/rebuild/ncn-rebuild-worker-nodes.sh ncn-w001
```
Comment (nrockershousen) [inline operations/node_management/Rebuild_NCNs.md:47]: command prompt should be `ncn-m001`

File: operations/node_management/Rebuild_NCNs.md [hunk 4]
```diff
@@ -42,8 +42,14 @@ ncn# echo $XNAME
...
+```
+* [Master node]
```
Comment (nrockershousen) [inline operations/node_management/Rebuild_NCNs.md:49]: The same 3 comments from above apply here

**Comment by nrockershousen** [comment]
Please update the TOC links for the NCN rebuild topics here: operations/index.md#node-management

File: operations/node_management/Rebuild_NCNs.md (hunk 3 above)
Comment (dzou-hpe) [reply operations/node_management/Rebuild_NCNs.md:47]: i am not sure i am following on this one

**Review by djlapenta — APPROVED** [review]

**Pull request merged**